apps/tutorial-servo/main.cpp

```diff
@@ -0,0 +1,27 @@
+#include "CANMsg.h"
+#include "TutorialServo.h"
+
+CAN can(CAN_RX, CAN_TX);
```

Comment:
i'd recommend using the CAN constructor to specify the CAN bus frequency. there's a macro for it that you can use.

Reply:
this means u dont need to call can.frequency(50000)

Reply:
done